app/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "cakephp/app",
+	"description": "CakePHP Application skeleton",
+	"type": "library",
+	"keywords": ["application", "cakephp"],
+	"homepage": "http://cakephp.org",
+	"license": "MIT",
+	"authors": [
+		{
+			"name": "CakePHP Community",
+			"homepage": "https://github.com/cakephp/cakephp/graphs/contributors"
+		}
+	],
+	"support": {
+		"issues": "https://github.com/cakephp/cakephp/issues",
+		"forum": "http://stackoverflow.com/tags/cakephp",
+		"irc": "irc://irc.freenode.org/cakephp",
+		"source": "https://github.com/cakephp/cakephp"
+	},
+	"require": {
+		"php": ">=5.3.0",
+		"ext-mcrypt": "*"
+	},
+	"require-dev": {
+		"phpunit/phpunit": "3.7.*",
+		"cakephp/cakephp": "~2.8",
+		"cakephp/cakephp-codesniffer": "^1.0.0"
```

Comment:
@markstory Should cake sniffer be required for devs on the app? I believe the app should be free to use anything... Probably can put it as suggested.